File: src/Sequence/core/SqLog.java
package Sequence.core;

import arc.util.Log;

public class SqLog {
    public static final String modPrefix = "[seq] ";

    public static void info(String text, Object... args) {
        Log.info(modPrefix + text, args);
    }

    public static void info(Object object) {
        info(String.valueOf(object), SqConst.emptyObjArr);
    }

    public static void err(String text, Object... args) {
        Log.err(modPrefix + text, args);
    }

    public static void err(Object object) {
        info(String.valueOf(object), SqConst.emptyObjArr);
    }

    public static void warn(String text, Object... args) {
        Log.warn(modPrefix + text, args);
    }

    public static void warn(Object object) {
        info(String.valueOf(object), SqConst.emptyObjArr);
    }
}

File: src/sequence/core/SqLog.kt
package sequence.core

import arc.util.Log

object SqLog {
    const val modPrefix = "[seq] "
    fun info(text: String, vararg args: Any?) = Log.info(modPrefix + text, *args)
    fun info(obj: Any) = info(obj.toString(), *SqVars.emptyObjArr)
    fun err(text: String, vararg args: Any?) = Log.err(modPrefix + text, *args)
    fun err(obj: Any) = info(obj.toString(), *SqVars.emptyObjArr)
    fun warn(text: String, vararg args: Any?) = Log.warn(modPrefix + text, *args)
    fun warn(obj: Any) = info(obj.toString(), *SqVars.emptyObjArr)
    fun debug(text: String, vararg args: Any?) = Log.debug(modPrefix + text, *args)
    fun debug(obj: Any) = debug(obj.toString(), *SqVars.emptyObjArr)
}
